apps/chat/tests/test_tools.py

```diff
@@ -150,3 +151,49 @@ def test_create_schedule_message_experiment_does_not_exist():
         ).count()
 
         assert scheduled_message_count == 0
+
+
+@pytest.mark.django_db()
+class TestDeleteReminderTool:
+    def _invoke_tool(self, session, **tool_kwargs):
+        tool = DeleteReminderTool(experiment_session=session)
+        return tool.action(**tool_kwargs)
+
+    @pytest.fixture()
+    def session(self, db):
+        return ExperimentSessionFactory()
+
+    @pytest.fixture()
+    def schedule_params(self):
```

Comment:
interesting testing strategy, what's the advantage of having these a fixture vs a static class property vs a property method?

Reply:
No advantage really. A static class property will also work

Reply:
I decided to rather [go with a static method](https://github.com/dimagi/open-chat-studio/pull/597/commits/daf8bc705980090890a6f819d5d5aec5fe4969ff). This might be more expected than a fixture